packages/typespec-ts/test/azureIntegration/azureArmLargeHeader.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { assert } from "chai";
+import AzureArmLargeHeaderClientFactory, {
+  AzureArmLargeHeaderClient,
+  getLongRunningPoller,
+  isUnexpected
+} from "./generated/azure/resource-manager/large-header/src/index.js";
+
+describe.only("Azure ARM Large Header Rest Client", () => {
+  let client: AzureArmLargeHeaderClient;
+
+  beforeEach(() => {
+    client = AzureArmLargeHeaderClientFactory({
+      endpoint: "http://localhost:3000",
+      allowInsecureConnection: true
+    });
+  });
+
+  const SUBSCRIPTION_ID_EXPECTED = "00000000-0000-0000-0000-000000000000";
+  const RESOURCE_GROUP_EXPECTED = "test-rg";
+  const LARGE_HEADER_NAME = "header1";
+  it("should handle large headers in LRO operations", async () => {
+    // Test LRO POST operation with large headers (> 6KB each, > 12KB total)
+    const initialResponse = await client
+      .path(
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Azure.ResourceManager.LargeHeader/largeHeaders/{largeHeaderName}/two6k",
+        SUBSCRIPTION_ID_EXPECTED,
+        RESOURCE_GROUP_EXPECTED,
+        LARGE_HEADER_NAME
+      )
+      .post({});
```

Comment:
why {}? can we remove this?

Reply:
It should be optional and we should fix this in spector side. Let's skip it this time.